### PR TITLE
add managed instance to recommended list

### PR DESCRIPTION
**Pull request by alanrenmsft**
meanwhile, slide in the auto updated package.json

### CODE REVIEW

File: extensions/schema-compare/package.json [hunk 1]
```diff
@@ -1,59 +1,64 @@
...
-      ]
-    }
-  },
-  "dependencies": {
-    "vscode-extension-telemetry": "0.0.18",
-    "vscode-nls": "^3.2.1"
-  },
-  "devDependencies": {
-    "@types/mocha": "^5.2.5",
+	"name": "schema-compare",
```
Comment (kburtram) [inline extensions/schema-compare/package.json]: do we know why this file keeps getting updated in unrelated PRs?  This the second time I've seen this and the changes appear to contain more than just reformatting.

File: product.json [hunk 2]
```diff
@@ -45,6 +45,7 @@
 		"Microsoft.cms",
 		"Microsoft.dacpac",
 		"Microsoft.import",
+		"microsoft.managed-instance-dashboard",
```
Comment (kburtram) [inline product.json]: casing of Microsoft is different than all the other extensions.  I'd be consistent if there's no reason not to be.

Comment (alanrenmsft) [reply product.json]: fixed

File: extensions/schema-compare/package.json (hunk 1 above)
Comment (alanrenmsft) [reply extensions/schema-compare/package.json]: actually, i wasn't aware of the formatting changes until i see it here. the change was for the __metadata section added by the build step, i didn't look into why, but seems inherited from vscode. all other extensions had been updated with this section.

**Pull request merged**